ouroboros-consensus/test/consensus-test/Test/Consensus/BlockchainTime/Simple.hs

```diff
@@ -339,7 +340,7 @@ testOverrideDelay systemStart slotLength maxClockRewind numSlots = do
       withWatcher
         "testOverrideDelay"
         ( knownSlotWatcher btime $ \slotNo -> do
-            atomically $ modifyTVar slotsVar (slotNo :)
+            atomically $ modifyTVar slotsVar $ force . (slotNo :)
```

Comment:
The alternative would be to use `uncheckedNewTVarM` to create `slotsVar`. I think that would be more in line with what we do in tests usually.

Reply:
yeah, I think I prefer this over `force`